AdyenCard/Components/Stored Card/StoredCardAlertManager.swift

```diff
@@ -37,6 +46,7 @@ internal final class StoredCardAlertManager: NSObject, UITextFieldDelegate, Loca
             textField.keyboardType = .numberPad
             textField.placeholder = ADYLocalizedString("adyen.card.cvcItem.placeholder", self?.localizationParameters)
             textField.accessibilityLabel = ADYLocalizedString("adyen.card.cvcItem.title", self?.localizationParameters)
+            textField.accessibilityIdentifier = "AdyenCard.StoredCardAlertManager.textField"
```

Comment:
its shows this:
```
"<AdyenCard.StoredCardAlertManager: 0x7fb4d7f1d2f0>"	
```
for some reason instead of `AdyenCard.StoredCardAlertManager`, not sure why it works in all other places

Reply:
It doesn't work from classes 

```swift
        let scope = (scopeInstance as? UIView)?.accessibilityIdentifier ?? String(describing: scopeInstance)
        return "\(scope).\(postfix)"
```

should be `String(describing: scopeInstance.type)`

Reply:
there is no `.type`, and tried also `.self`, have the same result